ui/tests/integration/components/form-field-test.js

```diff
@@ -521,15 +525,15 @@ module('Integration | Component | form field', function (hooks) {
     assert.ok(spy.calledWith('myfield', ['baz', 'foo', 'bar']), 'onChange called with correct args');
   });
 
-  test('it renders: editType=checkboxList / possibleValues - with passed label, subtext, helptext, doclink', async function (assert) {
+  test('it renders: editType=checkboxList / possibleValues - with passed label, subtext, help text, doclink', async function (assert) {
     await setup.call(
       this,
       createAttr('myfield', '-', {
         editType: 'checkboxList',
         possibleValues: ['foo', 'bar', 'baz'],
         label: 'Custom label',
         subText: 'Some subtext',
-        helpText: 'Some helptext',
+        helpText: 'Some help text',
```

Comment:
[though] I know that in English "subtext" is a single word, while "help text" is two words, but since the properties are written as `subText` (with capital `T`) and `helpText` (also with capital `T`) I made the decision to use everywhere `helptext` to avoid questions/doubts of when "help text" should be two words and one word. If you feel strongly about it, that's OK, but otherwise I've found it much easier to keep it the same everywhere.